src/js/profile/chart.js

```diff
@@ -9,6 +9,7 @@ import XLSX from 'xlsx';
 import Papa from 'papaparse';
 
 import embed from "vega-embed";
+import { calculatePosition, DEFAULT_OPTIONS, formatValue } from "vega-tooltip";
```

Comment:
I don't think we need `DEFAULT_OPTIONS` or `formatValue`

Reply:
removed